packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_create_loan_scheme.test.ts

```diff
@@ -0,0 +1,71 @@
+import { GenesisKeys } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import { fundEllipticPair, sendTransaction } from '../test.utils'
+import { WIF } from '@defichain/jellyfish-crypto'
+import { BigNumber } from '@defichain/jellyfish-json'
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+
+const container = new LoanMasterNodeRegTestContainer()
+let providers: MockProviders
+let builder: P2WPKHTransactionBuilder
+
+beforeAll(async () => {
+  await container.start()
+  await container.waitForWalletCoinbaseMaturity()
+
+  providers = await getProviders(container)
+  providers.setEllipticPair(WIF.asEllipticPair(GenesisKeys[GenesisKeys.length - 1].owner.privKey))
+  builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic)
+
+  // Prep 1000 DFI Token for testing
+  await container.waitForWalletBalanceGTE(1001)
+})
+
+afterAll(async () => {
+  await container.stop()
+})
+
+beforeEach(async () => {
+  await container.waitForWalletBalanceGTE(11)
+
+  // Fund 10 DFI UTXO
+  await fundEllipticPair(container, providers.ellipticPair, 10)
+  await providers.setupMocks() // required to move utxos
+})
+
+it('should createLoanScheme', async () => {
+  const script = await providers.elliptic.script()
+  const txn = await builder.loans.createLoanScheme({
+    ratio: 200,
+    rate: new BigNumber(2.5),
+    identifier: 'scheme',
+    update: new BigNumber(1629104836999999999)
+  }, script)
+
+  // Ensure the created txn is correct.
+  // @TODO I face error here
+  await sendTransaction(container, txn)
```

Comment:
This line throw me error: Error: DeFiDRpcError: 'LoanSchemeTx: Cannot find existing loan scheme with id scheme (code 16)', code: -26